var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -13,6 +16,8 @@ class Cmake(Package):
     url      = 'https://github.com/Kitware/CMake/releases/download/v3.15.5/cmake-3.15.5.tar.gz'
     maintainers = ['chuckatkins']
 
+    executables = ['cmake']
```

Comment:
I'll try tomorrow to play a bit with this package and see how easy it is to detect variants based on `ccmake` and other executables being found.